aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -74,6 +82,10 @@ func resourceAwsApiGatewayRestApiCreate(d *schema.ResourceData, meta interface{}
 		params.BinaryMediaTypes = expandStringList(binaryMediaTypes.([]interface{}))
 	}
 
+	if minimumCompressionSize := d.Get("minimum_compression_size").(int); minimumCompressionSize > -1 {
```

Comment:
Nitpick: Do you mind putting the assignment on a separate line above as we're not checking result of casting? (which is AFAIK the common and often only reasonable use-case for these if one-liners)